app/components/footer.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+export default function Footer(){
+    return(
+        <footer className="bottom-0 w-full justify-center p-4 bg-black ">
```

Comment:
flexないからjustifyきかないかも
```suggestion
        <footer className="bottom-0 w-full p-4 bg-black ">
```